docs/imports/rock-the-vote.md

```diff
@@ -0,0 +1,120 @@
+
+We've added `referral=true` to the link so that we can know to not attribute the registration to the NS ID that is present in the URL. In this case, this NS ID is the referrer and not the registrant.
+
+If the referral column has `referral=true` in it, then proceed with the logic with dealing w/ non-member registrants above.
+
+### How to count these as impact
+
+Based on the above statuses, some should be counted as a RB and some should not. This determination was made by the executive team and allows us to report internally progress towards the organization's report back goal. Here's what counts as a reportback from the RTV export:
+
+- `register-form`
+- `register-OVR`
+
+Note: `register-form` and `register-OVR` are the only statuses that count as _registrations_.
+
+Rogue will NOT store this information, but will return a derived value in the JSON response when the voter registration post is created or read that holds this information. The logic to determine this is as follows:
+
+```php
+if (in_array($rogueStatus, ['confirmed', 'register-form', 'register-OVR'])) {
+    $reportbackStatus = 'T';
+} else {
+    $reportbackStatus = 'F';
+}
+```
+
+## Notes
+
+- Data uses the post `details` to determine `source` and `source_detail` used in voter registration reporting.
+- The `submission_created_at` date is when the importer ran. Details about when the registration was created/updated are in the `source_details`.
+- All of these signups will have a `source` of `importer-client` (this is how messaging is suppressed in C.io)
+- In early iterations of the import, the month that the registration came in would inform the `action` column (e.g., february-2018-rockthevote)
+- In early iterations of the import, we would pass Campaign/Run IDs as parameters within the referral code to use when upsert a  `voter-reg` post.
+- If a user shares their UTM'ed URL with other people, there could be duplicate referral codes but associated with different registrants:
+  See a [screenshot](https://cl.ly/0v210N283y2X) of what this data looks like (note: the user depicted in this spreadsheet is fake.)
+
+## Open Questions
+
+1. How can we distinguish between TV and RTV import? Do we need to?
+2. RTV has more birthdays in it, can we use that on the NS profile when we create accounts? (TV didn't provide this)
+3. We've been having some Chompy validation issues w/ the TV import -- is this the good time to tackle those things? If so, is a running list the best way of communicating some of the things we've seen in the referral column? There's one specific referral that's different and it's just `ads` -- these are from Google ads and the only way they could set up tracking that Google likes. Would love to talk through best way to deal with these...if it's something unique!
```

Comment:
I'm curious about some of the history here, there's some more context about this in the VR Tech inventory doc... seems related to https://www.pivotaltracker.com/story/show/170504511

Reply:
same with this link!

Reply:
I don't know if we need to do much more on our end here -- if I remember correctly, this was an issue with a bad process for creating voter reg links and people were using different formats/leaving out critical information. I'm pretty sure we've locked down this process now and the referral column is much cleaner and follows a format that they can more easily split the referral info into separate columns for analysis. In my opinion, we shouldn't be handling splitting that info on our end.

One thought, and I'm not sure if it's helpful, is if it'd be useful for Tej/others doing analysis to store referral information somewhere other than it's being stored now...

Reply:
per my chat with Tej last week, I think the ideal is to be able to filter voter registrations in Looker by the `source` and `source_detail` we include in a referral code.

Reply:
Removed this from open questions, as that work is referred to in https://www.pivotaltracker.com/story/show/170504511